tools/bug-report/pkg/bugreport/flags.go

```diff
@@ -0,0 +1,163 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bugreport
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+
+	config2 "istio.io/istio/tools/bug-report/pkg/config"
+	"istio.io/pkg/log"
+)
+
+var (
+	startTime, endTime, configFile, tempDir string
+	included, excluded                      []string
+	commandTimeout, since                   time.Duration
+	gConfig                                 = &config2.BugReportConfig{}
+)
+
+func addFlags(cmd *cobra.Command, args *config2.BugReportConfig) {
+	// k8s client config
+	cmd.PersistentFlags().StringVarP(&args.KubeConfigPath, "kubeconfig", "c", "", bugReportHelpKubeconfig)
```

Comment:
should be handled by istioctl already

Reply:
we need this for the standalone binary, let's see how that thread pans out, i'll keep this open till then.